MediaBrowser.Model/Net/MimeTypes.cs

```diff
@@ -12,6 +12,10 @@ namespace MediaBrowser.Model.Net
     /// <summary>
     /// Class MimeTypes.
     /// </summary>
+    ///
+    /// http://en.wikipedia.org/wiki/Internet_media_type
```

Comment:
Maybe this should be put in a `<remarks>` tag?

Reply:
Updated